[2022-05-13 17:20:19,424] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-05-13 17:20:19,435] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-05-13 17:20:19,435] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 17:20:19,435] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2022-05-13 17:20:19,435] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 17:20:19,478] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2022-05-11 00:00:00+00:00
[2022-05-13 17:20:19,481] {standard_task_runner.py:52} INFO - Started process 10348 to run task
[2022-05-13 17:20:19,487] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2022-05-11T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', '/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpjla2_t6c', '--error-file', '/tmp/tmp0wggk7q3']
[2022-05-13 17:20:19,487] {standard_task_runner.py:80} INFO - Job 48: Subtask also_run_this
[2022-05-13 17:20:19,540] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-11T00:00:00+00:00 [running]> on host roshan-Latitude-E5440
[2022-05-13 17:20:19,619] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-05-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-11T00:00:00+00:00
[2022-05-13 17:20:19,620] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-13 17:20:19,621] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2022-05-11T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-11 00:00:00+00:00: scheduled__2022-05-11T00:00:00+00:00, externally triggered: False>"']
[2022-05-13 17:20:19,626] {subprocess.py:85} INFO - Output:
[2022-05-13 17:20:19,627] {subprocess.py:92} INFO - run_id=scheduled__2022-05-11T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-11 00:00:00+00:00: scheduled__2022-05-11T00:00:00+00:00, externally triggered: False>
[2022-05-13 17:20:19,627] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-13 17:20:19,679] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220511T000000, start_date=20220513T115019, end_date=20220513T115019
[2022-05-13 17:20:19,737] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-13 17:20:19,762] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
